src/components/navbar/navbar.component.tsx

```diff
@@ -94,6 +108,25 @@ const Navbar: React.FC<NavbarProps> = ({
                 >
                   <UserAvatarFilledAlt20 />
                 </HeaderGlobalAction>
+                <HeaderGlobalAction
+                  aria-label="app switcher"
```

Comment:
These things are for screen readers and they should be filled out carefully.

`aria-label` -> `Toggle App Menu`

`aria-labelledby` -> ??? where is "app swithcer Icon" (note your typo) defined? This is referring to some ID. If there is none then don't use it.